packages/search/addon/components/navi-search-bar.hbs

```diff
@@ -1 +1,13 @@
-{{yield}}
+{{! Copyright 2020, Yahoo Holdings Inc. Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms. }}
+<BasicDropdown @tagName="div" @matchTriggerWidth={{true}} @verticalPosition="below" @class="navi-search" ...attributes as |dd|>
+  <div class="navi-search-bar">
+    <Input @type="text" @value={{this.searchQuery}} @placeholder="Search" @class="navi-search-bar__input"
+      @keyUp={{fn this.onKeyUp dd}} @focusIn={{fn dd.actions.open}} data-ebd-id="{{dd.uniqueId}}-trigger" />
+    <NaviIcon @icon="search" class="navi-search-bar__icon" />
+  </div>
+  {{#if this.searchResults}}
+    <dd.content @class="navi-search-results navi-search-results--box navi-search-result-wrapper slide-fade">
+      <NaviSearchResult::Wrapper @searchResults={{this.searchResults}} @closeResults={{fn dd.actions.close event}} class="navi-search-results__item navi-search-results--box"/>
```

Comment:
Here unless I pass an extra argument the dropdown doesn't close, I don't really understand why.

Reply:
What happens if you just have fn in front but no arg

Reply:
Nothing happens, you navigate to the next page but the dropdown doesn't close with no warning or error in the console.